cf_mssql_broker_config.json

```diff
@@ -32,7 +32,7 @@
 	"brokerMssqlConnection": {
 		"server":   "localhost\\sqlexpress",
 		"database": "master",
-		"driver": 	"sql server",
+		"driver": 	"{SQL Server Native Client 11.0}",
```

Comment:
We should document the dependency. In the case of having a broker that connects to a remote SQL Server, this dependency should already be available on the broker host machine.